### PR TITLE
Remove tenant admin login, fix table alignment, style issue

### PR DESCRIPTION
**Pull request by ritawatson**
## Describe the Change

- This PR removes "... as a tenant admin" from the Palette login step in VM Management docs, as VM actions are not restricted to tenant admins:
[Create the VM Dashboard](https://deploy-preview-1309--docs-spectrocloud.netlify.app/vm-management/vm-packs-profiles/create-vm-dashboard-profile)
[Enable the Dashboard](https://deploy-preview-1309--docs-spectrocloud.netlify.app/vm-management/vm-packs-profiles/enable-vm-dashboard)
[Deploy VM from a template](https://deploy-preview-1309--docs-spectrocloud.netlify.app/vm-management/create-manage-vm/standard-vm-operations/deploy-vm-from-template)
[Update VM Configuration](https://deploy-preview-1309--docs-spectrocloud.netlify.app/vm-management/create-manage-vm/standard-vm-operations/update-vm-configuration)
[Migrate a VM](https://deploy-preview-1309--docs-spectrocloud.netlify.app/vm-management/create-manage-vm/standard-vm-operations/migrate-vm-to-different-node)
[Take a Snapshot](https://deploy-preview-1309--docs-spectrocloud.netlify.app/vm-management/create-manage-vm/standard-vm-operations/take-snapshot-of-vm)
[Clone a VM](https://deploy-preview-1309--docs-spectrocloud.netlify.app/vm-management/create-manage-vm/standard-vm-operations/clone-vm)
[Create a VM template](https://deploy-preview-1309--docs-spectrocloud.netlify.app/vm-management/create-manage-vm/create-vm-template)
[Access virtctl](https://deploy-preview-1309--docs-spectrocloud.netlify.app/vm-management/create-manage-vm/access-cluster-with-virtctl)

- The PR also fixes alignment in the [VM User Roles and Permissions](https://deploy-preview-1309--docs-spectrocloud.netlify.app/vm-management/vm-roles-permissions) table where bullet lists caused ragged margins in the Restrictions and Palette columns.

- Fixes occurrences of "drop-down menu" to meet our internal style. Previously, the menu name was boldfaced along with "drop-down menu".

## Review Changes

💻 [Add Preview URL]()

🎫 [Jira Ticket]()

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1309#" title="Author self-approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by karl-cardenas-coding — APPROVED** [review]

**Pull request merged**
